sentences/en/_common.yaml

```diff
@@ -17,6 +17,8 @@ responses:
     no_entity: "Sorry, I am not aware of any device called {{ entity }}"
     no_entity_in_area: "Sorry, I am not aware of any device called {{ entity }} in the {{ area }} area"
     no_entity_in_floor: "Sorry, I am not aware of any device called {{ entity }} in the {{ floor }} floor"
+    entity_wrong_state: "Sorry, no device is in the expected state"
+    feature_not_supported: "Sorry, no device supports the required features"
```

Comment:
These feel very clunky as responses. I wouldn't have the slightest clue what I can do to fix anything if I tried to pause playback and I got back a "Sorry, no device is in the expected state".

Any chance they can be overridden at sentence/intent level with some `metadata` and keep these just as defaults? I feel like "Sorry, no device is currently playing" sounds much clearer.

Reply:
Hmmm, I could slip the expected state in there. I'll need to translate it, of course.
The features is more of a problem, as there don't seem to be strings for them anywhere I can find.